package.json

```diff
@@ -30,8 +30,7 @@
     "build": "tsdx build",
     "build:storybook": "build-storybook",
     "lint": "tsdx lint",
-    "test": "tsdx test --passWithNoTests",
-    "prepare": "tsdx build"
```

Comment:
We should probably move this over to `prepublishOnly` instead of removing it outright so it will only run before we publish a new version. [(reference)](https://docs.npmjs.com/misc/scripts)

Reply:
Good idea 👍